samsawan/SendBird-JavaScript
# PL-4 : Align  Member List

# Things Done
- Removed the Javascript that wrote CSS styling on the fly.
- Alphabetically ordered the imports in widget.js
- Alphabetically ordered the exports in consts.js (no removals or logic changes)
- Removed drop-shadow because now it's unnecessary as the boxes are aligned and shadow is hidden
- Added pixel sizes to align the items

# How To Test
- npm start as usual
- see the pretty things
- if you see things that aren't pretty, test has failed.

# Related PRs
